sdk/cognitiveservices/cognitiveservices-translatortext/package.json

```diff
@@ -1,57 +0,0 @@
-{
-  "name": "@azure/cognitiveservices-translatortext",
```

Comment:
Thanks for deleting this library. @xirzec Do we have a protocol for patching deleted libraries? I didn't find a release tag to checkout the source code from for last release or any release for that matter. I wonder if we should make one more release first to register such a tag and also to include a deprecation note in its readme?

Also, could you please open an issue to deprecate this [package on npm](https://www.npmjs.com/package/@azure/cognitiveservices-translatortext) once the new one is released?

Reply:
Should the old code remain and be deleted after the lib is deprecated?

Reply:
Shall the ticket be created in our ADO or what would be the best place for it?

Reply:
> Should the old code remain and be deleted after the lib is deprecated?

The code is needed in cases we need to make a release, patching ones mostly. If we delete it now, it will be hard to retrieve it later on such a release is needed. I am suggesting to make a release now to create a release tag in our repo so we can checkout the source code from in case we need to make a release in the future.

> Shall the ticket be created in our ADO or what would be the best place for it?

I think creating it here in the github repo is fine but feel free to duplicate it elsewhere.

Reply:
Who would do the release? I am not even sure when/how this old SDK was created.

Reply:
Let me get back to you on this.

Reply:
Delete was reverted.

Reply:
Since this library has very low usage and hasn't released since 7/2020, I think we can safely deprecate with a pointer to the new package. It's unlikely we would patch it given that it depends on the soon-to-be deprecated ms-rest-js.

A simple issue on this GitHub would suffice for deprecation, I will work creating a new issue type to make it a little easier.

Reply:
Tracked in https://github.com/Azure/azure-sdk-for-js/issues/25544